### PR TITLE
PUBD-142: Add DirectManageURL functionality to the JSchol CMS

### PR DESCRIPTION
**Pull request by hardyoyo**
PUBD-142: Add two new props/attrs directManageURLauthor and directManageURLeditor, define existing logic as default, and use new attrs if they exist

### CODE REVIEW

File: app/socksMysql.rb [hunk 1]
```diff
@@ -52,12 +52,27 @@ def service(ready)
...
   #################################################################################################
   def bidiTransfer(localSock, remoteSock)
+    first = true
     while true
       rds, wrs, ers = IO.select([localSock, remoteSock], [])
       rds.each { |r|
         data = r.recv(8192)
         data.empty? and return
-        #puts "Tranferring data from #{r == localSock ? "local" : r == remoteSock ? "remote" : r}: #{data.inspect}"
+        if first
```
Comment (myucekul) [inline app/socksMysql.rb:61]: Can we make this change separately to have the change history more accurately reflect the reason for the change.

Comment (hardyoyo) [reply app/socksMysql.rb:61]: sure, I'll untangle this

File: app/jsx/components/WizardInertComp.jsx [hunk 2]
```diff
@@ -25,7 +25,10 @@ class HeaderComp extends React.Component {
 class ManageSubmissionsComp extends React.Component {
   render() {
     let submit_url = "https://submit.escholarship.org"
-    let editor_url = this.props.type == 'campus' ? submit_url : submit_url+"/subi/directAdmin?target="+this.props.unit_id
+    let default_editor_url = this.props.type == 'campus' ? submit_url : submit_url+"/subi/directAdmin?target="+this.props.unit_id
```
[the remark this answers is not in the document]
Comment (hardyoyo) [reply app/jsx/components/WizardInertComp.jsx:28]: unknown, this is existing JSchol logic, just moved to a "default" variable

File: app/socksMysql.rb (hunk 1 above)
Comment (hardyoyo) [reply app/socksMysql.rb:61]: untangled: https://github.com/eScholarship/jschol/pull/625

**Pull request merged**